angular2/src/angular2TextMask.ts

```diff
@@ -33,12 +33,10 @@ export class MaskedInputDirective implements OnInit, AfterViewInit, ControlValue
   private _onTouched = () => {}
   private _onChange = (_: any) => {}
 
-  private isInit: boolean = false
-
   constructor(private renderer: Renderer, private element: ElementRef) {}
 
   ngAfterViewInit() {
-    !this.isInit && this.setupMask()
+    !this.textMaskInputElement && this.setupMask()
```

Comment:
I personally do not like this inline check with the function call. I would prefer a real **if** with a statement afterwards. Makes things more readable